internal/k3d/registry.go

```diff
@@ -22,3 +35,62 @@ func (cl *RegistryList) Unmarshal(data []byte) error {
 	cl.Registries = registries
 	return nil
 }
+
+type Config struct {
+	MappedRegistryPort string `yaml:"mapped_registry_port"`
+}
+
+func SaveRegistryPort(port string) error {
```

Comment:
Please add a short Godoc style comment about the purpose of this function (here and below)